src/django/contricleaner/lib/parsers/data_parser.py

```diff
@@ -0,0 +1,16 @@
+from typing import List
+from abc import ABC, abstractmethod
+
+
+class DataParser(ABC):
```

Comment:
Data parser isn't good name for this.
1. it doesn't parse data. so data is wrong name
2. it doesn't actually parse.

Let's leave it as is and rename it as separate PR.